app/views/layouts/_navbar.html.erb

```diff
@@ -10,7 +10,8 @@
             <li class="nav-item">
                 <%= link_to (fa_icon 'sticky-note', text: 'Notes'), notes_path, class: 'nav-link' %>
             </li>
-            <li class="nav-item"> <%= link_to (fa_icon 'users', text: "My Contacts"), "/contacts", class: 'nav-link' %> </li>
+          <li class="nav-item"> <%= link_to (fa_icon 'users', text: "My Contacts"), "/contacts", class: 'nav-link' %> </li>
+           <li class="nav-item"> <%= link_to (fa_icon 'search', text: "Search"), "/users/search", class: 'nav-link' %> </li>
```

Comment:
why is this an extra tab in the navbar? I thought it would be a field in the normal home/index?

Reply:
for me this extra view does not provide so much more information on the first thought

Reply:
it was specified like this in the User Story